apiv2/tests.py

```diff
@@ -276,3 +279,61 @@ def test_target_valid(self):
         form = SoundCombinedSearchFormAPI(data={'target': target})
         self.assertTrue(form.is_valid())
         self.assertEqual(form.cleaned_data['target'], target)
+
+
+class TestSoundListSerializer(TestCase):
+
+    fixtures = ['licenses', 'sounds']
+
+    def setUp(self):
+        self.ss = Sound.objects.all()[0:5]
+        self.sids = [s.id for s in self.ss]
+        self.factory = RequestFactory()
+
+    def test_num_fields(self):
+        # Test that serializer returns only fields included in fields parameter of the request
+
+        sounds_dict = Sound.objects.dict_ids(sound_ids=self.sids)
+
+        # When 'fields' parameter is not used, return default ones
+        dummy_request = self.factory.get(reverse('apiv2-sound-text-search'), {'fields': ''})
+        serialized_sound = SoundListSerializer(list(sounds_dict.values())[0], context={'request': dummy_request}).data
+        self.assertItemsEqual(serialized_sound.keys(), DEFAULT_FIELDS_IN_SOUND_LIST.split(','))
```

Comment:
is this just luck that these things are ordered in the same way? I think when I do similar tests in dunya I wrap each side in sorted()

Reply:
`assertItemsEqual` works with unordered lists so it works here :)